File: paradicms_etl/namespaces/worksheet.py
from rdflib import URIRef
from rdflib.namespace import DefinedNamespace, Namespace


class WORKSHEET(DefinedNamespace):
    _fail = True
    _NS = Namespace("http://www.paradicms.org/ns/worksheet#")

    # Properties
    featureSet: URIRef

    # Resources
    Feature: URIRef
    FeatureSet: URIRef

File: gui/app/worksheet/src/ts/vocabularies/WORKSHEET.ts
import {Namespace} from "@paradicms/vocabularies";

export class WORKSHEET {
  static readonly NS = Namespace("http://www.paradicms.org/ns/worksheet#");

  // Properties
  static readonly featureSet = WORKSHEET.NS("featureSet");

  // Resources
  static readonly Feature = WORKSHEET.NS("Feature");
  static readonly FeatureSet = WORKSHEET.NS("FeatureSet");
}
